api/api.proto

```diff
@@ -13,6 +13,9 @@ service Engine {
     rpc ListDrivers(ListDriversRequest) returns (ListDriversResponse) {}
 
     // SQL stuff.
+    // Ping connection.
+    rpc PingConnection(PingRequest) returns (PingResponse) {}
```

Comment:
not sure if we need to export special grpc methods for it.
Pinging mysql is just `db.exec("SELECT 1")` which can be done by client code using existing `SQL` method.

Reply:
This was also discussed [here](https://github.com/src-d/minutes/pull/453/files#diff-8cf96c0828de9b8b2e9cf2c633e939edR60), but I haven't actually studied this solution before the meeting.

If I understood correctly @smacker you propose to do something as:
```
stream, err := client.SQL(ctx, &api.SQLRequest{Query: "SELECT 1"})
```
instead of exposing an ad-hoc method as currently in the PR.

Anyway I came across [this](https://stackoverflow.com/questions/3668506/efficient-sql-test-query-or-validation-query-that-will-work-across-all-or-most#answer-3670000), hence your solution will surely work, but the client is now aware that on the server we're using a `SELECT 1`-compatible database (mysql in our case). Even if changing the underlying db to another one which is not `SELECT 1`-compatible is highly improbable, I think that from a design perspective making the client agnostic from the db being used by the server is much cleaner. By decoupling this logic we can also avoid modifying the client if tomorrow we want to migrate to another non-`SELECT 1`-compatible db.

WDYT?

Reply:
we do rely on mysql and to be more specific on gitbase that isn't 100% mysql. I think it's okay.

If everybody else agree to keep new methods, I'm okay with it.
I just think it's an unnecessary complication of the code base and API.

Reply:
@carlosms @dpordomingo wdyt?

Reply:
I agree with @smacker; source{d} Engine is already tied to gitbase, so `SELECT 1` LGTM

Reply:
Ok! Gonna change this then 👍